app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -218,6 +218,9 @@ def render_unavailable
         message = I18n.t("curation_concerns.workflow.unauthorized")
         respond_to do |wants|
           wants.html do
+            if @presenter.blank?
```

Comment:
@cam156 that's right. So we need a SuppressedPresenter. The ShowPresenter is inappropriate as we're not showing the full representation